src/analyzer/protocol/ssl/proc-certificate.pac

```diff
@@ -3,10 +3,18 @@ function proc_certificate(is_orig: bool, certificates : bytestring[]) : bool
 	if ( certificates->size() == 0 )
 		return true;
 
+	// this has to execute in both contexts, ssl and tls-handshake. In one we have flipped_,
+	// in the other we have ssl_analyzer()->GetFlipped(). And in both cases the other case
+	// does not work (and cannot be made to work easily).
+
+#ifndef USE_FLIPPED
+	bool flipped_ = zeek_analyzer()->GetFlipped();
+#endif
+
 	zeek::ODesc common;
 	common.AddRaw("Analyzer::ANALYZER_SSL");
 	common.Add(zeek_analyzer()->Conn()->StartTime());
-	common.AddRaw(is_orig ? "T" : "F", 1);
+	common.AddRaw(is_orig ^ flipped_ ? "T" : "F", 1);
```

Comment:
Fancy. :-) I'm going back and forth if I would find method/function `IsClient()` more readable, as this `^` construct is used quite a bit. Can't decide, so I'll just note it.